scripts/06_from_JoshuaBandong.py

```diff
@@ -0,0 +1 @@
+Hello Hello from Joshua Bandong
```

Comment:
Please enclose the code in a `print` function, for example:

```suggestion
print("Hello Hello from Joshua Bandong")
```